### PR TITLE
Completed html forms exercise.

### PR DESCRIPTION
**Pull request by randygerada**
Screen Shot 2022-10-11 at 2 11 53 PM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Forms!</h1>
+    <form action="http://www.example.com/">
+      <div>
+        <label for="email">Email</br></label>
+        <input type="text" name="email">
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure the input has an `id` which matches the `for` attribute on the label

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      </div>
+      <div>
+        <label for="choices">Why is Coding cool?</br></label>
+        <select name="choices">
```
Comment (BrettAlbright) [inline html-forms/index.html]: same as above comment

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          <option value="fun">Because its fun</option>
+          <option value="powerful">Because its powerful</option>
+        </select>
+      </div>
+    </br>
+      <div>
+        <label for="favorite-language">Favorite Language (choose one)</br></label>
+      </br>
+        <input type="radio" name="language" value="HTML"/
+        checked="HTML">
```
Comment (BrettAlbright) [inline html-forms/index.html]: this would need to be attached to the previous line

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        HTML
+        <input type="radio" name="language" value="CSS" />
```
Comment (BrettAlbright) [inline html-forms/index.html]: each of these inputs needs their own label

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        CSS
+        <input type="radio" name="language" value="Javascript" />
+        Javascript
+      </div>
+    </br>
+      <div>
+        <label for="information">Would you like to sign up for more information?</br></label>
+      </br>
+        <input type="checkbox" name="subscribe" value="Sign me up!"/>
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure to add the proper id here.

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        Sign me up!
+      </div>
+      <div>
+        <label for="comments">Comments</br></label>
+        <textarea name="comments" cols="30" rows="10"></textarea>
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure to add the proper id here

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good! Just a few small updates

**Comment by randygerada** [comment]
Screen Shot 2022-10-11 at 9 23 33 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**